sched/sched/sched_note.c

```diff
@@ -116,24 +117,35 @@ static unsigned int g_note_disabled_irq_nest[CONFIG_SMP_NCPUS];
  ****************************************************************************/
 
 static inline void sched_note_flatten(FAR uint8_t *dst,
-                                      FAR void *src, size_t len)
+                                      FAR void *src_, size_t len)
 {
+  FAR uint8_t *src = src_;
+
   switch (len)
```

Comment:
Code is fine, but I was just thinking why just not to:
```
switch (len)
  {
    case 8:
    case 4:
    case 2:
    case 1:
#ifdef CONFIG_ENDIAN_BIG
      {
        FAR uint8_t *end = (FAR uint8_t *)src + len - 1;
        while (len-- > 0)
          {
            *dst++ = *end--;
          }
      }
#else
      memcpy(dst, src, len);
#endif
      break;
    default:
      DEBUGASSERT(FALSE);
      break;
  }
```
Seems to be more clear.

Reply:
Done